PURL-TYPES.rst

```diff
@@ -253,6 +253,23 @@ github
       pkg:github/package-url/purl-spec@244fd47e07d1004
       pkg:github/package-url/purl-spec@244fd47e07d1004#everybody/loves/dogs
 
+githubactions
+------
+``githubactions`` for GitHub Actions:
```

Comment:
What is the default `repository_url`? Does this apply to Actions? If not, why not? The purl spec does not differentiate between public Internet facing repositories and internal ones. If there's a default `repository_url`, even an internal one, it should be part of the purl type definition.

Reply:
Sure, I could add one of those if you think it's helpful. GitHub Actions are always in practice pulled from the current GitHub instance you're on. So if you're on github.com, it's that, and if you're on an enterprise server instance, it's that.